### ISSUE TITLE
Need a saga that triggers when a sled joins the cluster or reboots

### ISSUE THREAD

**Issue by jmpesp** [oxidecomputer/omicron#2545]
There needs to be actions performed when a new sled joins the cluster (or when an existing sled reboots). At minimum, OPTE v2p mappings may need to be created or restored.

**Comment by davepacheco** [comment]
I assume the work required here has since been done, since we've done a lot of sled reboots and sled adds at this point.  @jmpesp if that's wrong can you reopen this?

**Comment by jmpesp** [comment]
Yeah, this is an outdated issue that is obviated by our existing background tasks.
